ui/shared/utils/constants.js

```diff
@@ -1684,51 +1684,65 @@ export const VARIANT_EXPORT_DATA = [
   },
 ]
 
-export const ALL_INHERITANCE_FILTER = 'all'
-export const RECESSIVE_FILTER = 'recessive'
-export const HOM_RECESSIVE_FILTER = 'homozygous_recessive'
-export const X_LINKED_RECESSIVE_FILTER = 'x_linked_recessive'
-export const COMPOUND_HET_FILTER = 'compound_het'
-export const DE_NOVO_FILTER = 'de_novo'
 export const ANY_AFFECTED = 'any_affected'
 
-export const INHERITANCE_FILTER_OPTIONS = [
-  { value: ALL_INHERITANCE_FILTER, text: 'All' },
+export const TOPMED_FREQUENCY = 'topmed'
```

Comment:
everything removed from this file was moved to the search page specific constant file. Everything added was in `shared/componenets/panel/search` constants file but was being used by non-search componenets and so I therefore moved them into this shared constants file instead